### DISCUSSION TITLE
[Announcement] Homework 2 is out

### DISCUSSION THREAD

**Discussion by KAIST-JongchanPark** [prosyslab-classroom/cs348-information-security#465] [Announcements]
Hi all,
this is an announcement about homework 2.

Here is the GitHub classroom link for this homework:
[https://classroom.github.com/a/KIxBKMIv](https://classroom.github.com/a/KIxBKMIv)

Due: 04. 05 (Wed.) 23:59:59
Late Due: 04. 07 (Fri.) 23:59:59 ([late submission penalty](https://github.com/prosyslab-classroom/cs348-information-security#homework)).

Thanks.

**Comment by KihongHeo** [comment]
Also submit your repo to Gradescope as before.
(with a funny nick name)
